benchmark_runner/common/grafana/update_grafana_latest_value_mappings.py

```diff
@@ -12,7 +12,7 @@ class UpdateGrafanaLatestValueMappings:
     """
     # Prevent error messages from being saved instead of versions in case of a connection issue
     MAX_VERSION_LEN = 20
-    LAST_ES_FETCH_DAYS = 30
+    LAST_ES_FETCH_DAYS = 14
```

Comment:
This change is unrelated and should be a separate PR if it's needed.

Reply:
Agree, remove